### ISSUE TITLE
underscore template compiler

### ISSUE THREAD

**Issue by prabirshrestha** [andrewdavey/cassette#269]
would be great if cassette includes inbuilt underscore template compiler. 

I haven't tried this out yet, but I found a .net implementation of underscore template compiler at https://github.com/chaliy/aspnet-underscore-bundler/blob/master/src/UnderscoreBundler/UnderscoreTemplateCompiler.cs

**Comment by jpsullivan** [comment]
I have to agree. This would definitely be a great feature to have in Cassette.

**Comment by prabirshrestha** [comment]
It might also be better to just embed underscore.js in Cassette. Then call the following method.

``` js
var compiledString = _.template('<div><%=name%></div>').source;
```

it will generate the following string

```
function(obj){ var __p='';var print=function(){__p+=Array.prototype.join.call(arguments, '')}; with(obj||{}){ __p+='
'+ (name)+ '
'; } return __p; }
```

**Comment by mshwery** [comment]
+1
